R/bm-read-file.R

```diff
@@ -2,35 +2,35 @@
 #'
 #' @section Parameters:
 #' * `source` A known-file id, or a CSV(?) file path to read in
-#' * `format` One of `c("parquet", "feather", "fst")`
+#' * `file_type` One of `c("parquet", "feather", "fst")`
 #' * `compression` One of the values: `r paste(known_compressions, collapse = ", ")`
-#' * `output` One of `c("arrow_table", "data_frame")`
+#' * `output_type` One of `c("arrow_table", "data_frame")`
 #'
 #' @export
-read_file <- Benchmark("read_file",
-  setup = function(source = names(known_sources),
+read_file <- Benchmark("file-read",
+  setup = function(source = c("fanniemae_2016Q4", "nyctaxi_2010-01"),
                    # TODO: break out feather_v1 and feather_v2, feather_v2 only in >= 0.17
-                   format = c("parquet", "feather"),
+                   file_type = c("parquet", "feather"),
                    compression = c("uncompressed", "snappy", "lz4"),
-                   output = c("arrow_table", "data_frame")) {
-    # format defaults to parquet or feather, but can accept fst as well
-    format <- match.arg(format, c("parquet", "feather", "fst"))
-    # the output defaults are retrieved from the function definition (arrow_table and data_frame)
-    output <- match.arg(output)
+                   output_type = c("arrow_table", "data_frame")) {
+    # file_type defaults to parquet or feather, but can accept fst as well
+    file_type <- match.arg(file_type, c("parquet", "feather", "fst"))
```

Comment:
We can still run it for comparison if we want, though AFAIK we don't have a policy on when we actually do run external packages and compare against them. It might be nice to run these when there's an Arrow release and include them for context in that report? Not priority, but maybe useful context.

Or was {fst} deprecated or something? I thought I remembered something like that, but now I don't see it.

Reply:
I think maybe you are thinking [disk.frame](https://cran.r-project.org/web/packages/disk.frame/readme/README.html). And I think the subtext here is that it really doesn't matter too much to include it so why remove. Convinced.

Reply:
Oh, yep, that was it! Thanks

Reply:
Yeah, we might need to drop support for fst when we move to datalogistik (or do something slightly different if we were to use it), but for now let's keep it here